Makefile

```diff
@@ -66,6 +66,12 @@ endif
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
+BUILTIN_CLOUD_PROVIDERS ?= aws
```

Comment:
azure ?

Reply:
That's odd, i added this, i'm not sure why you didn't see it on your end